app/services/EventReportService.scala

```diff
@@ -187,21 +189,58 @@ class EventReportService @Inject()(eventReportConnector: EventReportConnector,
     }
   }
 
-  def submitEventDeclarationReport(pstr: String, userAnswersJson: JsValue)
-                                  (implicit headerCarrier: HeaderCarrier, ec: ExecutionContext, request: RequestHeader): Future[JsValue] = {
-    eventReportConnector.submitEventDeclarationReport(pstr, userAnswersJson).map(_.json)
+  private def validatePayloadAgainstSchema(payload: JsObject, schemePath: String, eventName: String)
```

Comment:
I think `schemePath` is a typo and should be `schemaPath`.

Reply:
Changed